README.md

```diff
@@ -1,81 +1,88 @@
 # DataBasic
 
-Learn Java and other software engineering practices by building a simple database.
-The goal of this exercise is to learn, and be confident in developing using:
+## REQUIREMENTS
 
-- Java
-- IntelliJ IDEA
-- JUnit
-- Git & GitHub
-- Markdown
-- Software Design
-- Clear technical communication
-- StackOverflow to find answers to questions
-- Library documentation
-- Javadoc
-- Unix command line basics
-- Object Oriented Design
-- JSON
-- Technical development practices
-  - Refactoring
-  - TDD
-  - Clean Code
-  - Clean Architecture
+1. Compatible only with *nix operating systems. 
+2. The software will run in CLI mode.
```

Comment:
Pro tip: In markdown, it doesn't matter which number you put first, it will always show up correctly. I tend to always use `1.`. The reason for this, is so that if I ever need to add a bullet point between 4 and 5, I don't have to rename 5, 6, 7, 8, etc. So:

```
1. Compatible only with *nix operating systems. 
1. The software will run in CLI mode.
1. (...)
```

Reply:
I understand and thanks for the tip. I have a question, about to add a bullet point between 4 and 5, you mean to change the number 4 or 5 for a bullet? Because just for insert a bullet between them I can do it using:

4. This
- blah
5. That 

Thanks!

Reply:
No no, I do mean a numbered list item, not an un-numbered bullet point :)
My bad.

Reply:
Ah yes, that would be a headache try to insert an numerate item there...